src/OverlapMesh.cpp

```diff
@@ -1883,21 +1901,22 @@ void GenerateOverlapMeshFromFace(
 
 			meshOverlap.vecSourceFaceIx.push_back(ixSourceFace);
 			meshOverlap.vecTargetFaceIx.push_back(ixCurrentTargetFace);
+			meshOverlap.vecFaceArea.push_back(dArea);
 		}
 	}
 }
 
 ///////////////////////////////////////////////////////////////////////////////
 
-void GenerateOverlapMesh_v2(
+void GenerateOverlapMeshKdx(
 	const Mesh & meshSource,
 	const Mesh & meshTarget,
 	Mesh & meshOverlap,
     OverlapMeshMethod method,
 	const bool fAllowNoOverlap,
     const bool fVerbose
 ) {
-#if defined(OVERLAPMESH_RETAIN_REPEATED_NODES)
+#if defined(OVERLAPMESH_USE_SORTED_MAP)
 	NodeMap nodemapOverlap;
 #endif
 #if defined(OVERLAPMESH_USE_UNSORTED_MAP)
```

Comment:
Isn't this either or type definition ? If for whatever reason you define both `OVERLAPMESH_USE_SORTED_MAP` and `OVERLAPMESH_USE_UNSORTED_MAP`, this will give a compiler error. And it is confusing in general for readability. Probably may be better to use a single macro that you can check with predefined macro int value like below:
```
#define OVERLAPMESH_USE_SORTED_MAP 0
#define OVERLAPMESH_USE_UNSORTED_MAP 1
#define OVERLAPMESH_NODEMAP OVERLAPMESH_USE_SORTED_MAP

#if OVERLAPMESH_NODEMAP == OVERLAPMESH_USE_SORTED_MAP
// 
#elif OVERLAPMESH_NODEMAP == OVERLAPMESH_USE_UNSORTED_MAP
//
#else
#endif
```